src/vision/status_quo/alan_creates_a_hanging_alarm.md

```diff
@@ -0,0 +1,131 @@
+# 😱 Status quo stories: Alan Creates a Hanging Alarm
+
+## 🚧 Warning: Draft status 🚧
+
+This is a draft "status quo" story submitted as part of the brainstorming period. It is derived from real-life experiences of actual Rust users and is meant to reflect some of the challenges that Async Rust programmers face today.
+
+If you would like to expand on this story, or adjust the answers to the FAQ, feel free to open a PR making edits (but keep in mind that, as they reflect peoples' experiences, status quo stories [cannot be wrong], only inaccurate). Alternatively, you may wish to [add your own status quo story][htvsq]!
+
+## The story
+
+[Alan] is a developer on the Bottlerocket project.
+[Bottlerocket] is a Linux-based open-source operating system that is purpose-built by Amazon Web Services for running containers.
+Alan created a rust program, [pubsys], to ensure that Bottlerocket update repositories are healthy.
+A _repository verification alarm_ uses pubsys to check the validity of Bottlerocket update repositories and notifies the team if any issues are found.
+
+### Multiple Tokio Runtimes
+
+Bottlerocket uses its own [tough] library to read and write TUF repositories.
+This library was created before async became widespread and [reqwest] changed its main interface to `async`.
+When reqwest switched to async, Alan used the `reqwest::blocking` feature instead of re-writing tough to be an async interface.
+(Maybe Alan [should](https://github.com/awslabs/tough/issues/213) make tough an async interface, but he hasn't yet.)
+In order to provide a non-async interface, `reqwest::blocking` creates a tokio runtime so that it can await futures.
+
+In pubsys Alan created some parallel downloading logic while using the above libraries.
+Without realizing the danger, he created a tokio runtime in pubsys and used futures/await to do this parallelization.
```

Comment:
Code examples would really help me to understand what pattern is causing the error.